ecs-logs-collector.sh

```diff
@@ -163,6 +163,9 @@ collect_brief() {
   get_docker_info
   get_docker_containers_info
   get_docker_logs
+  get_docker_sysconfig
+  get_docker_sysconfig_storage
```

Comment:
These steps seem like they could be merged together into a `get_docker_sysconfig` as they're both primarily concerned with "legacy" configuration methods. We added backwards compatibility to allow users to continue with these methods, so the collection could be skipped on systems where that isn't the case.

The current approach is also fine by me, so I'll leave it up to you to decide :+1:

Reply:
Merged into a single function 👍